File: JavaStudies/imccalculator.java

public class imccalculator {
	public static void main(String[] args) {
		double weight = 100.5;
		double height = 1.75;
		double imc;
		
		imc = (weight/(height*height));
		
		System.out.println("O seu IMC é de: " + imc);
		
		if(imc < 18.5){
			System.out.println("Você está abaixo do peso");
		}
		if(imc >= 18.6 && imc <= 24.9) {
			System.out.println("Seu peso é ideal!");
		}
		if(imc >= 25 && imc <= 29.9) {
			System.out.println("Sobrepeso");
		}
		else {
			System.out.println("Obesidade");
		}
	}

}

File: PythonStudies/imcCalculator.py
# Estudo IND
# IMC Calculator
# Insira seu peso e altura e calcule seu Indice de Massa corporal!


print('Vamos calcular seu IMC?')
weight = float(input('Insira aqui seu peso em Kg: '))
height = float(input('Insira sua altura em metros: '))

imc = weight / (height * height)

p = ('Seu IMC é de: ' + str(imc))
print(p)

if(imc < 18.5):
  print('Você está abaixo do peso')
if (imc >= 18.6 and imc <= 24.9):
  print('Você está no peso ideal')
if (imc >= 25 and imc <= 29.9):
  print('Sobrepeso')
if (imc  > 29.9):
  print('Obesidade')